app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,51 @@
+class OrdersController < ApplicationController
+  include LineItemsHelper
+  before_action :find_order, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @orders = Order.all
+  end
+
+  def show
+    @find = Order.find_by(id: params[:id])
+  end
+
+  def new
+    if current_cart.line_items.empty?
```

Comment:
thử
`return redirect_to("/", notice: "Your cart is empty") if current_cart.line_items.empty?`
xem